tests/acceptance/test_update.py

```diff
@@ -67,33 +73,36 @@ def test_too_big_image_update(self):
             return
 
         # Make a too big update
-        run("dd if=/dev/zero of=image.dat bs=1M count=0 seek=2048")
-        output = run('mender -rootfs image.dat ; echo "ret_code=$?"')
+        subprocess.Popen("dd if=/dev/zero of=image.dat bs=1M count=0 seek=1024", shell=True).wait()
+        subprocess.Popen("artifacts write rootfs-image -t %s -i test-update-too-big -u image.dat -o image-too-big.mender" % image_type, shell=True).wait()
+        put("image-too-big.mender", remote_path="/var/tmp/image-too-big.mender")
+        output = run("mender -rootfs /var/tmp/image-too-big.mender ; echo 'ret_code=$?'")
+
 
         assert(output.find("no space left on device") >= 0)
         assert(output.find("ret_code=0") < 0)
 
-    def test_network_based_image_update(self, image_dat):
+    def test_network_based_image_update(self):
         http_server_location = pytest.config.getoption("--http-server")
         bbb = pytest.config.getoption("--bbb")
 
         if not env.host_string:
             # This means we are not inside execute(). Recurse into it!
-            execute(self.test_network_based_image_update, image_dat)
+            execute(self.test_network_based_image_update)
```

Comment:
This will probably break local testing unless you create `image.dat` manually, but that's alright, I can fix that later (I have some related work to do in that area anyway).